hackathon_site/dashboard/frontend/src/components/orders/OrdersFilter/OrderFilter.tsx

```diff
@@ -0,0 +1,181 @@
+                    checked={
+                        field.value ? field.value.includes(item.id.toString()) : false
+                    }
+                />
+                <Chip
+                    size="small"
+                    label={item.numOrders}
+                    className={styles.filterCategoryChip}
+                />
+            </div>
+        ))}
+    </FormGroup>
+);
+
+const OrderFilter = ({ handleReset, handleSubmit }: FormikValues) => {
+    const status = [
+        {
+            id: 1,
+            status: "Pending",
+            numOrders: 2,
+        },
+        {
+            id: 2,
+            status: "Ready for Pick up",
+            numOrders: 4,
+        },
+        {
+            id: 3,
+            status: "Checked out",
+            numOrders: 4,
+        },
+    ];
+
+    return (
+        <div className={styles.filter}>
+            <Paper elevation={2} className={styles.filterPaper} square={true}>
+                <form onReset={handleReset} onSubmit={handleSubmit}>
+                    <fieldset>
+                        <legend>
+                            <Typography variant="h2">Order by</Typography>
+                        </legend>
+                        <Field
+                            name="ordering"
+                            component={RadioOrderBy}
+                            options={orderByOptions}
+                        />
+                    </fieldset>
+                    <Divider
+                        className={styles.filterDivider}
+                        data-testid="orderFilterDivider"
+                    />
+                    <fieldset>
+                        <legend>
+                            <Typography variant="h2">Status</Typography>
+                        </legend>
+                        <Field
+                            name="status"
+                            component={CheckboxCategory}
+                            options={status}
+                        />
+                    </fieldset>
+                </form>
+            </Paper>
+            <div className={styles.filterBtns}>
+                <Button
+                    type="reset"
+                    color="secondary"
+                    onClick={handleReset}
+                    data-testid="clear-button"
+                >
+                    Clear all
+                </Button>
+                <Button
+                    type="submit"
+                    onClick={handleSubmit}
+                    color="primary"
+                    variant="contained"
+                    fullWidth={true}
+                    className={styles.filterBtnsApply}
+                    disableElevation
+                    data-testid="apply-button"
+                >
+                    Apply
+                </Button>
+            </div>
+        </div>
+    );
+};
+
+export const EnhancedOrderFilter = () => {
+    const onSubmit = ({ ordering, status }: any) => {
```

Comment:
this shouldn't be any type, you should create an interface with types for ordering and status in this file and use it here.